examples/react-dashboard/src/App.js

```diff
@@ -232,7 +233,14 @@ function Example() {
 
 export default function App() {
   return (
-    <RoomProvider id={roomId} initialPresence={{ cardId: null, cursor: null }}>
+    <RoomProvider
+      id={roomId}
+      initialPresence={{
+        selectedDataset: { cardId: null, dataKey: null },
+        cursor: null,
+        cardId: null,
```

Comment:
Is this top-level `cardId` field even used anywhere?

Reply:
yes it is used here https://github.com/liveblocks/liveblocks/blob/fd6b98461b945646dd3dcceb64d9c0dbcab3e4fb/examples/react-dashboard/src/components/Card.js#L63